### PR TITLE
pulls in PR changes (manual) from https://github.com/komand/plugin-sd…

### PR DESCRIPTION
**Pull request by mikeatlas-r7**

### CODE REVIEW

**Comment by mikeatlas-r7** [comment]
@mrinehart-r7 your blessing again, sir?

Also I don't seem to have the same pep8 settings as you, since I get a lot of warnings for things your pep8 isn't complaining about...

I'm running vscode with pep8 default extension.

**Comment by mrinehart-r7** [comment]
@mikeatlas-r7 Oh, I'm getting a hell of a lot of PEP8 warnings. There's just too many PEP8 to tackle for a PR for this stuff. I just fixed them for the file you touched since there weren't many.

File: komand/__init__.py [hunk 1]
```diff
@@ -39,9 +40,14 @@ def where():
             # to do their own verifications against the bundle.
             return env_var
         else:
+            logging.warn('certifi.where(): Using old certifi value %s', old_certifi_value)
```
Comment (mrinehart-r7) [inline komand/__init__.py]: `warn` is deprecated, we should be using `warning` instead

File: komand/__init__.py [hunk 2]
```diff
@@ -39,9 +40,14 @@ def where():
...
             return old_certifi_value
     except Exception as ex:
+        logging.warn('certifi.where(): Using old certifi value %s', old_certifi_value)
```
Comment (mrinehart-r7) [inline komand/__init__.py]: Same as above

Comment (mikeatlas-r7) [reply komand/__init__.py]: `fix warnings (for deprecated warn->warnings)` META META META

**Comment by mikeatlas-r7** [comment]
requesting one more review @mrinehart-r7

**Comment by mrinehart-r7** [comment]
kk

**Comment by mrinehart-r7** [comment]
LGTM

**Pull request merged**